pkgs/top-level/aliases.nix

```diff
@@ -231,6 +231,7 @@ mapAliases ({
   clang_10 = throw "clang_10 has been removed from nixpkgs"; # Added 2024-01-26
   clang_11 = throw "clang_11 has been removed from nixpkgs"; # Added 2023-01-24
 
+  cq-editor = throw "cq-editor has been revomed, as it use a dependency that was disabled since python 3.8 and was last updated in 2021";
```

Comment:
```suggestion
  cq-editor = throw "cq-editor has been removed, as it use a dependency that was disabled since python 3.8 and was last updated in 2021";
```